helm/hpcc/templates/_helpers.tpl

```diff
@@ -1929,7 +1930,9 @@ spec:
      {{- end }}
   dnsNames:
      {{- range $dnsName := $local.dnsNames }}
-  - {{ (printf "%s.%s" $dnsName $domain) | quote }}
+     {{- range $altDomain := $local.allDomains }}
```

Comment:
If I were being super picky I would say the variable name "altDomain" is not right.  $local.allDomains contains the primary domain and the alts.  But hardly matters.